src/context.rs

```diff
@@ -131,7 +144,7 @@ impl<A: Actor> Context<A> {
     /// Stop all actors on this address
     fn stop_all(&mut self) {
         if let Some(strong) = self.ref_counter.upgrade() {
-            strong.0.store(false, Ordering::Release)
+            strong.0.0.store(false, Ordering::Release)
```

Comment:
Shouldn't this (and probably somewhere else too) call shared_drop_notifier and set it as dropped? Otherwise, am I just missing where it's called here?

Reply:
The `DropNotifier` gets triggered when it's dropped - hence the name.
So once all references to the `shared_drop_notifier` are gone, the `DropNotifier` is dropped and the corresponding `DropNotice`s get notified automatically - no need to call stop or anything. I find this approach much safer & easier to use. There's no way one could forget to call it and no need to keep an extra counter or anything.

Reply:
Oh, cool, I missed that, sorry!